### PR TITLE
Use viridis cmap for doc images

### PR DESCRIPTION
**Pull request by dkillick**
A follow-on to #2889, which didn't quite do the trick...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -116,8 +116,12 @@ script:
   # which should be investigated further in the future.
   - >
     if [[ $TEST_TARGET == 'doctest' ]]; then
+      MPL_RC_DIR=$HOME/.config/matplotlib;
+      mkdir -p $MPL_RC_DIR;
+      echo 'backend : Qt4Agg' > $MPL_RC_DIR/matplotlibrc;
```
Comment (pelson) [inline .travis.yml]: This should be ``agg``, right?

**Comment by pelson** [comment]
Thanks @dkillick!

File: .travis.yml (hunk 1 above)
Comment (dkillick) [reply .travis.yml]: If we only need to save images, then yes!

**Comment by pelson** [comment]
Worked a treat. Thanks for doing this @dkillick!

https://scitools-docs.github.io/iris/master/gallery.html

**Pull request merged**